salt/modules/postgres.py

```diff
@@ -1465,10 +1465,21 @@ def is_available_extension(name,
 
 def _pg_is_older_ext_ver(a, b):
     '''Return true if version a is lesser than b
-    TODO: be more intelligent to test versions
-
+    
+    Compare versions by breaking the version number into
+    its major, minor, and patch components.
     '''
-    return a < b
+    a, b = a.split('.'), b.split('.')
```

Comment:
why not using distutils.version?

Reply:
I was unaware of distutils.version. That's a good suggestion. Thank you.

Reply:
Or even `salt.utils.versions.LooseVersion` would perhaps work here as well.